tests/agent_features/test_dimensional_metrics.py

```diff
@@ -70,37 +83,88 @@ def _test():
 
 
 @pytest.mark.parametrize("tags,expected", _test_tags_examples)
+@reset_core_stats_engine()
 def test_record_dimensional_metrics_inside_transaction(tags, expected):
-    @validate_transaction_metrics("test_record_dimensional_metrics_inside_transaction", background_task=True, dimensional_metrics=[("Metric/1", expected, 1), ("Metric/2", expected, 1)])
+    @validate_transaction_metrics(
+        "test_record_dimensional_metrics_inside_transaction",
+        background_task=True,
+        dimensional_metrics=[("Metric.1", expected, 1), ("Metric.2", expected, 1)],
+    )
     @background_task(name="test_record_dimensional_metrics_inside_transaction")
     def _test():
-        record_dimensional_metrics([("Metric/1", 1, tags), ("Metric/2", 1, tags)])
+        record_dimensional_metrics([("Metric.1", 1, tags), ("Metric.2", 1, tags)])
 
     _test()
 
 
 @pytest.mark.parametrize("tags,expected", _test_tags_examples)
 @reset_core_stats_engine()
 def test_record_dimensional_metrics_outside_transaction(tags, expected):
-    @validate_dimensional_metrics_outside_transaction([("Metric/1", expected, 1), ("Metric/2", expected, 1)])
+    @validate_dimensional_metrics_outside_transaction([("Metric.1", expected, 1), ("Metric.2", expected, 1)])
     def _test():
         app = application_instance()
-        record_dimensional_metrics([("Metric/1", 1, tags), ("Metric/2", 1, tags)], application=app)
+        record_dimensional_metrics([("Metric.1", 1, tags), ("Metric.2", 1, tags)], application=app)
```

Comment:
Was the `/` to `.` conversion intentional here?

Reply:
Yeah I was having trouble querying these metrics in the UI with a slash. Not sure why but NRQL doesn't seem to accept it.